functions/go/annotate-mutations/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.16-alpine3.13
+ENV CGO_ENABLED=0
+WORKDIR /go/src/
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN go build -o /usr/local/bin/function ./
+
+#############################################
+
+FROM alpine:3.13
```

Comment:
do we need alpine or can we use distroless?

Reply:
I followed the existing functions' docker patterns